scripts/templates/selectorsyncset.yaml

```diff
@@ -0,0 +1,16 @@
+
+apiVersion: hive.openshift.io/v1alpha1
+kind: SelectorSyncSet
+metadata:
+  generation: 1
+  name: osd-manged-cluster-config
+  labels:
+    osd-config: True 
```

Comment:
```suggestion
    managed.openshift.io/osd: "true"
```

Reply:
Also were we not going to put the URL to the git repo?  Hash is going to be a little bit of a pain to put in, but would be nice.  Not required.

```
    managed.openshift.io/git-source: "https://github.com/openshift/managed-cluster-config/"
```

Reply:
is `managed.openshift.io/osd: "true"` to generic ? Maybe `managed.openshift.io/osd-config: "true"`

Reply:
Created the labels pr our conversation